tests/components/updater/test_init.py

```diff
@@ -59,11 +57,15 @@ def test_new_version_shows_entity_after_hour(
         async_fire_time_changed(hass, dt_util.utcnow() + timedelta(hours=1))
         yield from hass.async_block_till_done()
 
-    assert hass.states.is_state(updater.ENTITY_ID, NEW_VERSION)
+    assert hass.states.is_state("binary_sensor.updater", "on")
+    assert (
+        hass.states.get("binary_sensor.updater").attributes["newest_version"]
+        == NEW_VERSION
+    )
```

Comment:
@MartinHjelmare The tests fail already here. How can I verify that the binary sensor is set up and working? As I said, I would catch the dispatcher signal, but how?

Reply:
Sorry for taking your time...

Reply:
`yield from hass.async_block_till_done()` after setting up the component. We must wait for the platform to be loaded before firing time passing and checking the state of the entity.

Reply:
Awesome! Thank you very much. I didn't expect it to be that easy.

Reply:
Local tests pass every time, but CI tests sometimes fail. Do you have another idea?